build_product

```diff
@@ -250,6 +257,14 @@ set_no_derivatives_options() {
 	fi
 }
 
+set_quick_options() {
+	if test "$_arg_quick" = on ; then
+		for chosen_product in "${_arg_product[@]}"; do
+			QUICK+="generate-ssg-${chosen_product}-ds.xml "
```

Comment:
Usage of `+=` is interesting in this context, and apparently correct. I would consider using arrays - you would need to declare an empty array first.

```suggestion
			QUICK+=( "generate-ssg-${chosen_product}-ds.xml" )
```

Reply:
Done